extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -324,12 +324,23 @@ public function getPage() {
 				'ns' => $title->getNamespace()
 			];
 
-			if ( $this->isSupportedByMercury( $title ) ) {
-				// Empty category pages are not known but contain article list
+			// handle cases like starwars.wikia.com/wiki/w:c:clashroyale:Tesla (interwiki links)
+			$interWikiUrl = '';
+			InterwikiDispatcher::getInterWikiaURL( $title, $interWikiUrl, '' );
```

Comment:
this returns true because this method is used as a hook, and it always returns true